Documentation updates for v3.1 (#3475)
geopm/geopm: docs/source/index.rst

optimize system hardware settings to achieve energy efficiency and/or
performance objectives.

----

Source Code
-----------

To get the latest source code clone the git repository:

.. code-block:: bash

    $ git clone https://github.com/geopm/geopm.git

.. image:: https://geopm.github.io/images/github-button.png
   :target: https://github.com/geopm/geopm
   :width: 200
   :alt: View on Github

----

With GEOPM, a system administrator can:
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> |:closed_lock_with_key:| Provide secure access to hardware telemetry and controls

Reviewer: ```suggestion
|:closed_lock_with_key:| Enable users with fine-grained access management
```

  - Grant per-user or per-group access to individual metrics and controls, even
    when their underlying interfaces do not offer fine-grained access control
|:safety_vest:| Provide safe access to hardware settings
  - Ensure that user-driven changes to hardware settings are reverted when the
    user's process session terminates

> |:building_construction:| Develop new plugins

Reviewer: ```suggestion
|:building_construction:| Extend the open-core framework
```

  - Develop your own platform-specific monitor and control interfaces through
    the extensible plugin architecture

With GEOPM, an end user can:
^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> |:microscope:| Gather hardware telemetry and set hardware configuration

Reviewer: ```suggestion
|:microscope:| Monitor telemetry and configure settings on the platform
```
